parse/ConditionPythonParser.cpp

```diff
@@ -788,6 +788,8 @@ void RegisterGlobalsConditions(boost::python::dict& globals) {
     globals["IsSource"] = condition_wrapper(std::make_shared<Condition::Source>());
     globals["IsTarget"] = condition_wrapper(std::make_shared<Condition::Target>());
     globals["IsRootCandidate"] = condition_wrapper(std::make_shared<Condition::RootCandidate>());
+    globals["IsAnyObject"] = condition_wrapper(std::make_shared<Condition::All>());
+    globals["NoObject"] = condition_wrapper(std::make_shared<Condition::None>());
```

Comment:
@o01eg OK?

Reply:
Yes, `NoEffect` is for the same.

Reply:
note that `Condition::None` and `Condition::NoOp` are different